Documentation/DataSources/PrivateWorkbooksDeprecation.md

```diff
@@ -1,10 +1,75 @@
 # Private Workbooks Deprecation
-The ability to save changes to existing private workbooks is going away by July 1st, 2021. You will still be able to access your private workbooks but any edit or save can only be saved as a Shared Workbook. 
+Workbooks were available in two modes – private and shared. Private workbooks were accessible only to the author and was the default save mode.  
 
-# How should you proceed?
-If you choose to migrate your private workbook to a shared one, you can go to your private workbook and click "Save As" and select the "Shared Report" option from the drop down. If you don't want others to have access to your workbook, you can always manually change the RBAC to deny access to others.
+Private Workbooks were stored as a different Azure resource type than Shared Workbooks. We are merging these different Azure resource types and making one single standard type. This will allow it to take advantage of all the existing and new functionality available in Azure Workbooks. This includes 
 
-# How do I know if my workbook is private or not?
-![Private vs Shared Workbook](../Images/PrivateVsSharedWB.png)
+- Shared Workbooks can be queried via Azure Resource Graph (ARG) 
+- Shared Workbooks can support top level ARM features like other resource types, including but not limited to 
+    - Tags 
+    - Activity Log / Change Tracking 
+    - Resource locks 
+- Shared Workbooks can support ARM templates  
+- Shared workbooks are searchable in browse blade  
+- Shared Workbooks can support "Bring your own storage" (BYOS) feature 
+- Shared Workbooks can be saved in region of your choice 
 
-One of the easiest ways is to identify a private workbook and shared workbook by their icon on the gallery view. The private workbook will have a blue icon with a person figure in the corner. The shared workbook will have a green icon with a stack figure in the corner.
+Private workbooks will be deprecated by **June 30, 2021** to simplify the consumption and sharing experience. There are two stages to private workbook deprecation.  
+
+1. First stage - Switch default save mode for workbooks to shared 
+    - Every new workbook is **now** saved as shared workbook by default. 
+    - We have removed the ability to create/ update private workbooks. 
+ 
+2. Second stage - Convert existing private workbooks to shared workbooks  
+    - <ins>**You**</ins> will need to convert your existing private workbooks to shared workbooks by
+        - Either using ‘save’ / ‘save as’ (make a copy)  
```

Comment:
nit: a bunch of this is using office "smart quotes" instead of just standard quotes